examples/cxx/request_response_dynamic_data/src/server.cpp

```diff
@@ -40,7 +43,7 @@ auto main() -> int {
 
     std::cout << "Server ready to receive requests!" << std::endl;
 
-    auto counter = 1;
+    uint counter = 1;
```

Comment:
```suggestion
    auto counter = 1U;
```

It seems when using the `U` suffix, the `std::min<uint64_t>` can be changed back to just `std::min`. The `U` suffix needs also to be added to `1000000` in line 55.